templates/kafka-env.sh.j2

```diff
@@ -3,3 +3,5 @@ KAFKA_JVM_PERFORMANCE_OPTS="-server -XX:MetaspaceSize=96m -XX:+UseG1GC -XX:MaxGC
 {% if (memory_heapsize|int) > 1 %}
 KAFKA_HEAP_OPTS="-Xmx{{ memory_heapsize }}g -Xms{{ memory_heapsize }}g"
 {% endif %}
+KAFKA_LOG4J_OPTS="-Dlog4j.configuration=file:{{ conf_dest }}/log4j.properties"
+{{ env_var }}
```

Comment:
Considering that we remove `env_var` from the `defaults/main.yaml` it is then possible to use ` | default(omit)` in order to keep the output clean.

e.g.:

```
{{ env_var | default(omit) }}
```

Reply:
Done. I deleted the var from defaults, changed the name and added `| default(omit)`